plugins/outputs/stomp/stomp.go

```diff
@@ -23,32 +23,38 @@ type STOMP struct {
 	NetConn   net.Conn
 	Stomp     *stomp.Conn
 	serialize serializers.Serializer
+	Log       telegraf.Logger `toml:"-"`
 }
 
 //Connect ...
 func (q *STOMP) Connect() error {
-	var err error
 	if q.SSL == true {
-		tlsConfig, _ := q.ClientConfig.TLSConfig()
-
+		tlsConfig, err := q.ClientConfig.TLSConfig()
```

Comment:
missing ssl settings to configure certificates.
You copied the settings for this into the readme, but didn't implement these:
```
     # tls_ca = "/etc/telegraf/ca.pem"
     # tls_cert = "/etc/telegraf/cert.pem"
     # tls_key = "/etc/telegraf/key.pem"
     # insecure_skip_verify = false
```